OPAL/br/src/main/resources/reference.conf

```diff
@@ -100,6 +100,42 @@ org.opalj {
 
           ]
         }
+
+        DefaultEntryPoints{
```

Comment:
Please use a name that makes clear the connection to Android. In general, it should be the same name (and package structure) as the analysis/ProjectInformationKey that uses it.